src/util/PinConfig.js

```diff
@@ -1,17 +1,26 @@
-import Const from './PinConst';
+import { COUNT_TYPES } from './PinConst';
 
-let guid = '';
-for (var i = 0; i < 12; i++) {
-    guid += Const.GUID_VARS.substr(Math.floor(Math.random() * 60), 1);
+/**
+ * Generates a random pseudo-GUID
+ * @return {string} GUID
+ */
+function generateGuid() {
+    const GUID_VARS = '0123456789ABCDEFGHJKLMNPQRSTUVWXYZ_abcdefghijkmnopqrstuvwxyz';
+    let guid = '';
+    for (var i = 0; i < 12; i++) {
+        guid += GUID_VARS.substr(Math.floor(Math.random() * 60), 1);
+    }
+    return guid;
 }
 
-export default {
-    guid: guid,
-    [Const.COUNT.BUTTON]: 0,
-    [Const.COUNT.FOLLOW]: 0,
-    [Const.COUNT.PIN_SMALL]: 0,
-    [Const.COUNT.PIN_MEDIUM]: 0,
-    [Const.COUNT.PIN_LARGE]: 0,
-    [Const.COUNT.PROFILE]: 0,
-    [Const.COUNT.BOARD]: 0
-};
\ No newline at end of file
+export const guid = generateGuid();
```

Comment:
The guid is used for all of our logging calls. If they travel to Pinterest from one of the widgets, the guid is placed as a query parameter and then we can link it back to the user/domain/widget they came from. Could definitely go in Constants. We could also move counts onto PinterestBase as a static value like you did with `initialized` and get rid of this whole file. :+1:  Not this PR, it's already too big. lol